restore.go

```diff
@@ -119,5 +124,22 @@ func downloadDependency(dep vendor.Dependency) error {
 		return err
 	}
 
+	// Check for for manifests in dependencies
+	man := filepath.Join(dst, "vendor", "manifest")
+	venDir := filepath.Join(dst, "vendor")
+	if _, err := os.Stat(man); err == nil {
+		m, err := vendor.ReadManifest(man)
+		if err != nil {
+			return fmt.Errorf("could not load manifest: %v", err)
```

Comment:
A failure to read a manifest should not be a fatal error, it might not be a gvt manifest